src/components/Project/ProjectHeader/GnosisSafeBadge.tsx

```diff
@@ -21,17 +32,90 @@ export function GnosisSafeBadge({ href }: { href: string }) {
     )
   }
 
-  return (
-    <Tooltip
-      placement="bottom"
-      title={
-        <Trans>
-          This project is owned by a Safe.{' '}
-          <Link href={href}>See transactions</Link>.
-        </Trans>
-      }
-    >
-      <Icon component={SafeIcon} />
-    </Tooltip>
-  )
+  function DefaultBadge() {
+    return (
+      <Tooltip
+        placement="bottom"
+        title={
+          <Trans>
+            This project is owned by a Safe.{' '}
+            <Link href={href}>View transactions</Link>.
+          </Trans>
+        }
+      >
+        <Icon component={SafeIcon} />
+      </Tooltip>
+    )
+  }
+
+  // Red dot that appears conditionally next to Safe badge when connected user is on multisig and has unsiged tranctions
+  function Notice() {
+    return (
+      <div
+        style={{
+          position: 'absolute',
+          backgroundColor: 'red',
```

Comment:
**suggestion** can we use whatever `red` color we have defined in our color pallete?